include/ddc/kernels/splines/bsplines_non_uniform.hpp

```diff
@@ -28,39 +28,49 @@ struct NonUniformBSplinesBase
 
 } // namespace detail
 
-/// NonUniformPointSampling specialization of BSplines
+/**
+ * The type of a non-uniform BSplines 1D basis.
+ *
+ * @tparam Tag The tag identifying the continuous dimension which supports the building of the BSplines.
+ * @tparam D The degree of the BSplines.
+ */
```

Comment:
I am not sure what the correct formatting should be for "BSplines" but I don't think it should be "BSplines if it doesn't refer to a class. Wikipedia and "The NURBS book" use "B-splines" it is an abbreviation of "basis spline" so I think the "-" makes sense.
```suggestion
/**
 * The type of a non-uniform 1D basis of B-splines.
 *
 * @tparam Tag The tag identifying the continuous dimension which supports the building of the B-splines.
 * @tparam D The degree of the B-splines.
 */
```
If you agree, then this change probably needs applying over both these files